### ISSUE TITLE
Fractured wrist

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5886]
**Preferred term label:** Fractured wrist

**Synonyms** Wrist fracture; Broken wrist; Carpal bone fracture

**Definition (free text, please give PubMed ID)** Fracture of the wrist bone(s)., PMID: 31103090

**Parent term (use hpo.jax.org/app)** Abnormality of the carpal bones HP:0001191; Bone fracture HP:0020110

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
